Redesigning the UI

alainm23 opened the pull request:
Adapt friends to the same user interface as tasks, thoughts?

![image](https://user-images.githubusercontent.com/33765137/69994095-25c21c80-151b-11ea-955d-5771a020b928.png)

cassidyjames commented:
If this direction is pursued, we should look at getting SplitWindow into Granite https://github.com/elementary/granite/pull/315

alainm23 commented:
that's great, this is just an idea of ​​how it could look. We will have to wait for SplitWindow to arrive to master to use it. 

![image](https://user-images.githubusercontent.com/33765137/70065834-b94a2a80-15b9-11ea-8c73-97630f8df4b3.png)

alainm23 commented:
I removed the add button and fixed the margins so that everything is aligned with the close button.

![image](https://user-images.githubusercontent.com/33765137/75549165-07c26600-59fd-11ea-9cc9-fc3c2837299b.png)
